docs/src/use/migrate-to-9.0.0.md

```diff
@@ -209,6 +210,17 @@ As announced in our [blog post](/blog/2023/10/flat-config-rollout-plans/), the t
 
 **Related Issues(s):** [#13481](https://github.com/eslint/eslint/issues/13481)
 
+## <a name="stricter-rule-tester"></a> Stricter `RuleTester` checks
+
+In order to aid in the development of high-quality custom rules that are free from common bugs, ESLint v9.0.0 implements several changes to `RuleTester`:
+
+1. **Suggestion messages must be unique.** Because suggestions are typically displayed in an editor as a dropdown list, it's important that no two suggestions have the same message. Otherwise, it's impossible to know what any given suggestion will do. This additional check runs automatically.
```

Comment:
```suggestion
1. **Suggestion messages must be unique.** Because suggestions are typically displayed in an editor as a dropdown list, it's important that no two suggestions for the same lint problem have the same message. Otherwise, it's impossible to know what any given suggestion will do. This additional check runs automatically.
```

Maybe this, to additionally clarify?